app/helpers/patient_helper.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 
 # Helper methods for the patient model
-module PatientHelper # rubocop:todo Metrics/ModuleLength
+module PatientHelper
+  # This list contains all of the same states listed in app/javascript/components/data.js
+  def state_names
+    @state_names ||= YAML.safe_load(File.read('lib/assets/state_names.yml'), [Symbol])
```

Comment:
Also - I'm assuming the `[Symbol]` argument determines how it is read but am having trouble finding docs for that. Could you point me towards where you read that can be used this way?

Reply:
The `[Symbol]` argument is basically telling the YAML loader that it is OK for it to load keys/attributes that are supposed to be symbols. By default, the safe loader does not read in symbol types.